docs/CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # What's New with Enterprise-NG
 
+## v9.5.0
```

Comment:
This seems like its incorrect, maybe from another branch?

Reply:
We are on 10.2.0 now.

Reply:
I branched from the old master branch, so have rebased to main and added a CHANGELOG message.